changelog.md

```diff
@@ -11,6 +11,7 @@
 * [1511](https://github.com/zeta-chain/node/pull/1511) - move ballot voting logic from `crosschain` to `observer`
 * [1783](https://github.com/zeta-chain/node/pull/1783) - refactor zetaclient metrics naming and structure
 * [1774](https://github.com/zeta-chain/node/pull/1774) - split params and config in zetaclient
+* [1831](https://github.com/zeta-chain/node/pull/1831) - split params and config in zetaclient follow up
```

Comment:
It also needs to be put under `#unreleased`

Reply:
Never mind, we just need to move v14: https://github.com/zeta-chain/node/pull/1839